src/todolist/settings.py

```diff
@@ -18,7 +18,7 @@
 # See https://docs.djangoproject.com/en/1.7/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "@e2(yx)v&tgh3_s=0yja-i!dpebxsz^dg47x)-k&kq_3zf*9e*"
+SECRET_KEY = os.getenv("SECRET_KEY", "default-secret-key")
```

Comment:
It's recommended to avoid using a default secret key in production. Ensure that the `SECRET_KEY` environment variable is set in production environments to enhance security.